src/plugins/analysis/file_system_metadata/code/file_system_metadata.py

```diff
@@ -270,3 +310,10 @@ def _get_tar_file_mode_str(file_info: tarfile.TarInfo) -> str:
 
 def _has_correct_type(mime_type: str) -> bool:
     return mime_type in ARCHIVE_MIME_TYPES + FS_MIME_TYPES
+
+
+def _filemode_str_to_int(filemode: str) -> int:
```

Comment:
This function has potential to be of common usability. Consider relocating it to its inverse counterpart, which is already a common function.

Reply:
Putting the function in the same place (`filter.py`) as `octal_to_readable()` is problematic, because that function is located in the frontend code and we should not import from there from a plugin (backend)

Reply:
I understand. Then leave it as is - for the time being.